src/pretix/presale/templates/pretixpresale/fragment_week_calendar.html

```diff
@@ -4,11 +4,11 @@
         <details class="weekday {% if day.events %}has-events{% else %}no-events{% endif %} {% if day.today %}today{% endif %}"
                 data-date="{{ day.date|date:"SHORT_DATE_FORMAT" }}" open>
             <summary>
-                <h3><span class="fa fa-fw" aria-hidden="true"></span> {{ day.day_formatted }}</h3>
+                <h3><span class="fa fa-fw" aria-hidden="true"></span> <time datetime="{{ day.date|date:"Y-m-d" }}">{{ day.day_formatted }}</time></h3>
```

Comment:
Somehow, these headlines are now much larger and even break into multiple lines, which causes a mess

![20210929_165924](https://user-images.githubusercontent.com/64280/135298816-41653110-5b4a-42fd-8d6f-3aaeec0a5790.png)

Reply:
I will look into it.

Reply:
fixed.